chpl/chpl-api/e2e/script/newman-run.js

```diff
@@ -118,13 +163,17 @@ const searchviewControllerTests = {
 const jobs = [
   cb => newman.run(acbControllerTests,cb),
   cb => newman.run(apiKeyControllerTests,cb),
-  cb => newman.run(certifiedProductControllerTests,cb),
+  cb => newman.run(certifiedProductControllerDataTests2015listings,cb),
+  cb => newman.run(certifiedProductControllerDataTests2014listings,cb),
+  cb => newman.run(certifiedProductControllerNoDataTests,cb),
   cb => newman.run(developerControllerDataTests,cb),
   cb => newman.run(developerControllerNoDataTests,cb),
+  cb => newman.run(jobControllerTests,cb),
   cb => newman.run(productsControllerTests,cb),
   cb => newman.run(productVersionControllerTests,cb),
   cb => newman.run(statusControllerTests,cb),
   cb => newman.run(testinglabControllerTests,cb),
+  cb => newman.run(schedulerControllerTests,cb),
```

Comment:
I dont see schedulerControllerTests added in this PR, not sure why is it added here?

Reply:
same as above comment